pkg/mkverification-raw-efi/verify

```diff
@@ -399,6 +399,14 @@ tpm2_pcrread >> "$REPORT/summary.log"
 
 cp "/root/etc/eve-release" "$REPORT"
 
+find /sys/kernel/iommu_groups/ -type l > "$REPORT/iommu_groups.out"
+
+watchdogs=$(find /dev -name "watchdog*" | grep -vw "/dev/watchdog")
```

Comment:
Are you doing this to exclude the case when there is a /dev/watchdog but no /dev/watchdog<N>?

If so, do we do that exclusion elsewhere in EVE like in spec.sh?
If it is consistently applied, then it still makes sense to document that this is the intent in a comment.

Reply:
I need help understanding the question. The goal here is to report if there is more than one watchdog available in the system because that causes problems for EVE. I noticed that apart from devices `/dev/watchdog[0-31]`, there is always a device `/dev/watchdog`. So, I must exclude `/dev/watchdog` in my search. Another way to do this is by throwing a warning if there are more than two devices.

Reply:
Additionally, I must check if more than one watchdog causes issues in the verification process.

Reply:
Ah - a comment stating that would make sense.
And it might make sense to use simpler code like
find /dev -name 'watchdog[0-9]'